README.md

```diff
@@ -58,6 +58,12 @@ If you still can't find it, check if it [bundles](http://www.typescriptlang.org/
 This is usually provided in a `"types"` or `"typings"` field in the `package.json`,
 or just look for any ".d.ts" files in the package and manually include them with a `/// <reference path="" />`.
 
+#### Typescript 2.7 and earlier
+
+Definitely Typed only tests packages on Typescript 2.8 and later, as of November 2019.
+If you're using Typescript 2.0 to 2.7, you can still try installing `@types` packages &mdash; the majority of packages don't use fancy new Typescript features.
+But there's no guarantee that they'll work.
+Packages that existed before November 2019 may have older versions that are explicitly marked compatible with older versions of Typescript; use the tag "ts2.5" for Typescript 2.5, for example.
```

Comment:
I could be wrong, but I think most people reading this README are current or prospective package authors, but this section is addressed to package consumers. Also, as a package consumer, if I weren’t familiar with our infrastructure, I wouldn’t know what you mean by `use the tag "ts2.5" for Typescript 2.5`. If it’s important, it might be worth it to explain how to check and use it:

<img width="215" alt="image" src="https://user-images.githubusercontent.com/3277153/68420213-b7bf4b00-0150-11ea-9877-77bc87728510.png">

```
npm install @types/react@ts2.5
```

Reply:
Hmm, you're right. I'll rethink whether this section even needs to be here.

Reply:
Well, I re-read the beginning of the section and it's targetted to consumers; the How Can I Contribute section is below.

I added the react example.